substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -84,6 +84,15 @@ pub mod weights;
 pub use pallet::*;
 pub use weights::WeightInfo;
 
+macro_rules! override_params {
```

Comment:
is macro necessary? can we just make it a function taking mut ref arguments?

Reply:
Yes please only macros if *really* needed. Its makes the code less maintainable otherwise.

Reply:
gotcha! thanks for the feedback, let me convert that to normal function

Reply:
Resolved in https://github.com/paritytech/polkadot-sdk/pull/3843/commits/64ca32d16f80afee195a57a10f9142ea1ccb1ad5